test/scenarios/support_archive/install.go

```diff
@@ -75,30 +74,25 @@ func testSupportArchiveCommand(testDynakube dynatracev1beta1.DynaKube) features.
 		result := executeSupportArchiveCommand(ctx, t, environmentConfig, "--stdout", testDynakube.Namespace)
 		require.NotNil(t, result)
 
-		zipReader, err := gzip.NewReader(result.StdOut)
+		zipReader, err := zip.NewReader(bytes.NewReader(result.StdOut.Bytes()), int64(result.StdOut.Len()))
+
 		require.NoError(t, err)
-		tarReader := tar.NewReader(zipReader)
 
 		requiredFiles := collectRequiredFiles(t, ctx, environmentConfig.Client().Resources(), testDynakube)
-
-		hdr, err := tarReader.Next()
-		for err == nil {
-			requiredFiles = assertFile(t, requiredFiles, *hdr)
-			hdr, err = tarReader.Next()
+		for _, file := range zipReader.File {
+			requiredFiles = assertFile(t, requiredFiles, *file)
 		}
 
-		require.Equal(t, io.EOF, err)
-
 		assert.Emptyf(t, requiredFiles, "Support archive does not contain all expected files.")
 		logMissingFiles(t, requiredFiles)
 		return ctx
 	}
 }
 
 func executeSupportArchiveCommand(ctx context.Context, t *testing.T, environmentConfig *envconf.Config, cmdLineArguments, namespace string) *pod.ExecutionResult { //nolint:revive
-	resources := environmentConfig.Client().Resources()
+	r := environmentConfig.Client().Resources()
```

Comment:
you could call it `envResources` if you don't want to shadow imports

Reply:
the problem I have with resources is, that I get a warning in my IDE. And normally I do not have warnings as they might point to real problems. Having to look though all warnings an tell which ones are bad and which not creates much too much mental load (for me).
If we want to have meaningful names use some that do not produce warnings (give it some more thought). I went with environmentResources in this case. Otherwise I would stick to r, ns, ctx or the other commonly used abbrevs.